.github/workflows/dash.yml

```
@@ -0,0 +1,171 @@
+
+    - name: Set review request
+      run: echo "request-review=1" >> $GITHUB_ENV
+      if: (github.event_name == 'issue_comment' && contains(github.event.comment.body, '/request-license-review')) || env.isDependabotPR
+    
+    - name: Set license check
+      run: echo "license-check=1" >> $GITHUB_ENV
+      if: (github.event_name == 'issue_comment' && contains(github.event.comment.body, '/license-check')) || env.isDependabotPR
+
+    - name: Process license-vetting request
+      if: (env.request-review || env.license-check) && (!env.isDependabotPR)
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const payload = await github.rest.repos.getCollaboratorPermissionLevel({
+            ...context.repo, username: context.actor
+          });
+          const userPermission = payload?.data?.permission;
+          let reaction = 'rocket'
+          if (!(userPermission == 'write' || userPermission == 'admin')) { // not a committer
+            // Not a committer -> abort workflow
+            core.setFailed("Only committers are permitted to request license vetting and " + context.actor + " isn't one.")
+            reaction = '-1'
+          }
+          // react on comment to give early feedback that the request was understood
+          await github.rest.reactions.createForIssueComment({
+            ...context.repo, comment_id: context.payload?.comment?.id, content: reaction
+          });
+
+    # By default the git-ref checked out for events triggered by comments to PRs is 'refs/heads/master'
+    # and for events triggered by PR creation/updates the ref is 'refs/pull/<PR-number>/merge'.
+    # So by default only the master-branch would be considered when requesting license-reviews, but we want the PR's state.
+    # Unless the PR is closed, then we want the master-branch, which allows subsequent license review requests.
+    - uses: actions/checkout@v3
+      # use default ref 'refs/pull/<PR-number>/merge' for PR-events and 'refs/heads/master' for comments if the PR is closed
+      if: github.event.issue.pull_request == '' || github.event.issue.state != 'open'
+      with:
+        submodules: ${{ inputs.submodules }}
+    - uses: actions/checkout@v3
+      with:
+        ref: 'refs/pull/${{ github.event.issue.number }}/merge'
+        submodules: ${{ inputs.submodules }}
+      if: github.event.issue.pull_request != '' && github.event.issue.state == 'open'
+    
+    - name: Check license vetting status (and ask for review if requested)
+      id: check-license-vetting
+      uses: eclipse-set/build/.github/actions/dash-license-check-action@main
+      with:
+        request-review: ${{ env.request-review }}
+        project-id: ${{ inputs.projectId }}
+        pom: ${{ inputs.pom }}
+      env:
+        GITLAB_API_TOKEN: ${{ secrets.gitlabAPIToken }}
+    
+    # Hide outdated License comments
+    - uses: int128/hide-comment-action@5aa4ad6cf762098ebdc5d3c046e9e1fb35fe5ce8 # v1
```

Comment full version tag.
```suggestion
    - uses: int128/hide-comment-action@5aa4ad6cf762098ebdc5d3c046e9e1fb35fe5ce8 # v1.23.0
```
Btw: 1.24.0 has been released today

---

Updated to 1.24.0